source/PluginDev/Assets/GooglePlayGames/BasicApi/IPlayGamesClient.cs

```diff
@@ -350,6 +350,15 @@ public interface IPlayGamesClient
         /// popups for achievements and other game services elements.</remarks>
         /// <param name="gravity">Gravity for the popup.</param>
         void SetGravityForPopups(Gravity gravity);
+
+        /// <summary>
+        /// Shows the Play Games Player Profile UI for a specific player.
+        /// </summary>
+        /// </remarks>
+        /// <param name="playerId">The playerId. It cannot be null. 
+        /// <param name="callback">
+        /// </param>
+        void ShowPlayerProfileUI(string playerId, Action<UIStatus> callback);
```

Comment:
It seems in other API we are using userId instead of playerId. Let's replace it here for consistency to userId